### ISSUE TITLE
Chirho list

### ISSUE THREAD

**Issue by jcowey** [Edelweiss/hgv#47]
I would like to have a complete list of the the occurences of 

`<g type="chirho"/>`

in the files of DDB_EpiDoc_XML

I made a few changes yesterday so please make sure the DDB you use is up-to-date.

I would like to have read out (in tsv or csv) of the relevant DDB_EpiDoc_XML file:

the TM number, ddb-hybrid idno, each line which has an occurrence of `<g type="chirho"/>`

e.g. https://github.com/papyri/idp.data/blob/master/DDB_EpiDoc_XML/bgu/bgu.17/bgu.17.2695.xml

https://github.com/papyri/idp.data/blob/master/DDB_EpiDoc_XML/bgu/bgu.17/bgu.17.2695.xml#L16

https://github.com/papyri/idp.data/blob/master/DDB_EpiDoc_XML/bgu/bgu.17/bgu.17.2695.xml#L18

https://github.com/papyri/idp.data/blob/master/DDB_EpiDoc_XML/bgu/bgu.17/bgu.17.2695.xml#L67

https://github.com/papyri/idp.data/blob/master/DDB_EpiDoc_XML/bgu/bgu.17/bgu.17.2695.xml#L73

https://github.com/papyri/idp.data/blob/master/DDB_EpiDoc_XML/bgu/bgu.17/bgu.17.2695.xml#L123

I would also please like the follow categories from the corresponding HGV_meta_EpiDoc file
https://github.com/papyri/idp.data/blob/master/HGV_meta_EpiDoc/HGV70/69766.xml

Content of `<origPlace>`
https://github.com/papyri/idp.data/blob/master/HGV_meta_EpiDoc/HGV70/69766.xml#L33
Content of `<origDate>`
https://github.com/papyri/idp.data/blob/master/HGV_meta_EpiDoc/HGV70/69766.xml#L34

Content of bibliography/illustrations
https://github.com/papyri/idp.data/blob/master/HGV_meta_EpiDoc/HGV70/69766.xml#L92

Content of figure
https://github.com/papyri/idp.data/blob/master/HGV_meta_EpiDoc/HGV70/69766.xml#L98

I can do this is a rough way with Textwrangler (about 2400 in total) but putting together the relevant material across the boundary of DDB / HGV is too difficult for me.

If you could do this as soon as possible that would be great. Thank you.

**Comment by Edelweiss** [comment]
- (HGV number)
- TM number
- ddb-hybrid idno
- line number
- line chirho
- origPlace
- origDate
- illustrations (biblio)
- figure (link)

**Comment by jcowey** [comment]
Yes, great. And you noticed that the HGV number is also necessary. I had forgotten.

**Comment by Edelweiss** [comment]
Leider können wir mit unserer Standardversion von Saxon (Saxon-HE 9.6.0.7J from Saxonica) keine line numbers aus dem Stammdokument extrahieren.

https://www.saxonica.com/html/documentation/functions/saxon/line-number.html

> Saxon extension functions are not available under Saxon-HE

Das heißt, ich kann zwar die  ```tei:lb/@n``` angeben, aber nicht (ohne größere Umstände) die line number der Datei auf github.

**Comment by Edelweiss** [comment]
Unbekannte HGV-Nummer:

114797

https://github.com/papyri/idp.data/blob/master/DDB_EpiDoc_XML/c.ep.lat/c.ep.lat.236.xml#L17

https://github.com/papyri/idp.data/tree/master/HGV_meta_EpiDoc/HGV115

**Comment by Edelweiss** [comment]
https://docs.google.com/spreadsheets/d/1FpWIhFKAkN09tRjFStTCIsmXi9-ZbTE9Np3oLQrj96c/edit?usp=sharing
